esp/services/ws_dali/ws_daliservice.cpp

```diff
@@ -594,3 +594,135 @@ bool CWSDaliEx::onGetSDSSubscribers(IEspContext& context, IEspGetSDSSubscribersR
     }
     return true;
 }
+
+bool CWSDaliEx::onDisconnectClientConnection(IEspContext& context, IEspDisconnectClientConnectionRequest& req, IEspResultResponse& resp)
+{
+    try
+    {
+        checkAccess(context);
+
+        const char* url = req.getURL();
+        if (isEmptyString(url))
+            throw makeStringException(ECLWATCH_INVALID_INPUT, "URL not specified.");
+
+        MemoryBuffer mb;
+        mb.append("disconnect").append(url);
+        getDaliDiagnosticValue(mb);
+
+        VStringBuffer result("DisconnectClientConnection called for %s.", url);
+        resp.setResult(result);
+    }
+    catch(IException* e)
+    {
+        FORWARDEXCEPTION(context, e, ECLWATCH_INTERNAL_ERROR);
+    }
+    return true;
+}
+
+bool CWSDaliEx::onUnlockSDSLock(IEspContext& context, IEspUnlockSDSLockRequest& req, IEspResultResponse& resp)
+{
+    try
+    {
+        checkAccess(context);
+
+        const char* connectionIdHex = req.getConnectionID();
+        if (isEmptyString(connectionIdHex))
+            throw makeStringException(ECLWATCH_INVALID_INPUT, "ConnectionID not specified.");
+
+        MemoryBuffer mb;
+        mb.append("unlock").append(strtoll(connectionIdHex, nullptr, 16)).append(req.getClose());
```

Comment:
I had to check this, because it's not clear from the dalidiag usage (not a new issue) that it expects a hex number.
I think worth updating the daliadmin usage text at the same time to note it expect the connectionID to be hex.